cost-analyzer/values.yaml

```diff
@@ -520,3 +520,15 @@ serviceAccount:
 #  productKey: # apply business or enterprise product license
 #    key: ""
 #    enabled: false
+  # alertConfigs:
```

Comment:
@AjayTripathy i feel like this should potentially be its own block. productConfigs is already overwhelming IMO. WDYT?

Reply:
Yeah, let's pull without to a separate alertConfigs block @itscalvinwang

Reply:
@AjayTripathy we already have notifications: https://github.com/kubecost/cost-analyzer-helm-chart/blob/9cd209fba3f24abde4a64aa4e759c63e8163a6a6/cost-analyzer/values.yaml#L24

Should we not be using that?

Reply:
@AjayTripathy @itscalvinwang thoughts?

Reply:
I'm supportive of this; the Slack webhook setting already exists there so it makes sense to me.

Reply:
sounds good to me.